hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/AbstractGenericRecord.java

```diff
@@ -14,11 +14,10 @@
  * limitations under the License.
  */
 
-package com.hazelcast.nio.serialization;
+package com.hazelcast.internal.serialization.impl;
```

Comment:
```suggestion
package com.hazelcast.internal.serialization.impl.genericrecord;
```

Reply:
Do you want me to move this to there, by creating a new package? I moved it to here, because InternalGenericRecord was here. Also, since this is impl package, it doesn't really matter what we do now, we can always change it later